src/main/kotlin/com/hubermjonathan/discord/mitch/MitchBot.kt

```diff
@@ -1,44 +1,30 @@
 package com.hubermjonathan.discord.mitch
 
-import com.hubermjonathan.discord.common.models.Command
 import com.hubermjonathan.discord.mitch.groups.GroupsFeature
 import com.hubermjonathan.discord.mitch.management.ManagementFeature
 import net.dv8tion.jda.api.JDABuilder
-import net.dv8tion.jda.api.entities.Guild
 import net.dv8tion.jda.api.requests.GatewayIntent
-import net.dv8tion.jda.api.requests.restaction.CommandListUpdateAction
 import net.dv8tion.jda.api.utils.ChunkingFilter
 import net.dv8tion.jda.api.utils.MemberCachePolicy
-import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 fun main() {
-    val logger: Logger = LoggerFactory.getLogger("mitch")
+    val logger = LoggerFactory.getLogger("mitch")
     val jda = JDABuilder
         .createDefault(MitchConfig.MITCH_TOKEN)
         .setChunkingFilter(ChunkingFilter.ALL)
         .setMemberCachePolicy(MemberCachePolicy.ALL)
         .enableIntents(GatewayIntent.GUILD_MEMBERS)
         .build()
         .awaitReady()
+
+    jda.purdudesGuild.updateCommands().complete()
```

Comment:
This may warrant a comment, maybe something like:
```suggestion
    // Clear the commands before loading features
    jda.purdudesGuild.updateCommands().complete()
```